knowledge-base/what-happens-during-an-execution-preconf.mdx

```diff
@@ -0,0 +1,67 @@
+---
+title: What Happens During An Execution Preconfirmation?
+sidebarTitle: What Happens During A Preconf?
+description: "Learn about what happens behind the scenes during an execution preconfirmation."
+---
+
+Let's say you're a searcher or solver looking to execute a complex MEV strategy on Ethereum involving multiple DeFi protocols.
+
+## Problem
+
+Currently, when you send a transaction bundle:
+- You define your maximum slippage tolerance for each trade (eg. 1%)
+- You broadcast your bundle to one or more block builders.
+- You wait about 12 seconds (1 block) or more for it to be included.
+
+During this time, you have no guarantee that:
+- Your entire bundle will be included in the next block.
+- The transactions won't revert due to changing conditions.
+- Other MEV actors won't push your transaction toward your slippage limit
+
+As a result of these factors:
+- You might miss your target prices.
+- You might pay gas for failed transactions.
+- Your bundle may not get included, which is especially problematic for single-shot opportunities in DeFi. Some arbitrage opportunities only exist for a very brief window, and missing this window means losing the entire opportunity.
+
+## Using an Execution Preconfirmation
+
+<img height="200" src="/images/preconf-process.svg" />
+
+Instead of blindly sending your transaction, you can use an execution preconf. Here's how:
+
+**Note**: In the near future, you'll be able to add your transaction payload directly to your bid, which streamlines the process even further.
+
+1. Setting up your mev-commit bidder node
+
+   - Follow the Quickstart guide to set up your mev-commit bidder node and fund your account.
```

Comment:
let's add links to docs, there's an easy way to link to pages within mintlify it's / something directory

Reply:
added link